components/library/http-parser/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Denys Rtveliashvili.  All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		http-parser
+COMPONENT_VERSION=	2.1
+COMPONENT_FMRI=		library/http-parser
+COMPONENT_CLASSIFICATION=System/Libraries
+COMPONENT_PROJECT_URL=	https://packages.qa.debian.org/h/http-parser.html
```

Comment:
Please, link to original URL , for example https://github.com/nodejs/http-parser